ElectronicObserver/Data/TsunDbSubmission/EnemyComp.cs

```diff
@@ -1,72 +1,68 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Text.Json.Serialization;
 using ElectronicObserver.Data.Battle.Phase;
-using Newtonsoft.Json;
 
-namespace ElectronicObserver.Data;
+namespace ElectronicObserver.Data.TsunDbSubmission;
 
 public class EnemyComp : TsunDbEntity
 {
-	protected override string Url => "";
+	protected override string Url => throw new NotImplementedException();
```

Comment:
Empty string is probably good enough here, I didn't know it's not empty by default.

Reply:
I think the throw is good too since this entity shouldn't be sent alone ?

Reply:
I guess that's fine then.